packages/cli/src/net/artifacts/SiteAssetArtifactsService/uploadSiteVersion.mts

```diff
@@ -26,17 +26,19 @@ export async function uploadZippedSiteAsset(
   thirdPartyAppRid: ThirdPartyAppRid,
   version: string,
   zipFile: ReadableStream | Blob | BufferSource,
+  token: string,
```

Comment:
These are all going to have to be removed when we move off these APIs but for now can you try and do some cleanup to make these handwritten clients a bit more consistent with the generated ones? For example, group the foundryUrl / token provider together and potentially define request interfaces